models/DataObject/Service.php

```diff
@@ -946,7 +946,10 @@ public static function getValidLayouts(Concrete $object)
         }
 
         foreach ($list as $customLayout) {
-            $resultList[$customLayout->getId()] = $customLayout;
+            if($customLayout instanceof ClassDefinition\CustomLayout) {
```

Comment:
doesn't belong to this PR, does it?

Reply:
you right this does not belong to this PR....my bad! it was miss :/

Reply:
But doesn't hurt, so i kept it.